static/css/fullcalendar.css

```diff
@@ -0,0 +1,1124 @@
+/*!
```

Comment:
So I dug into this. We can indeed use a second gem source in the Gemfile, with the correct syntax. However, I can't find a way to load from gem assets rather than `/static`. There is a [jekyll-assets](https://github.com/jekyll/jekyll-assets) plugin but that brings us back to the discussion about plugins - even then, I'm not 100% sure it can load from gems.

I guess there's a reason jquery is already in /static...